### DISCUSSION TITLE
ISSUE

### DISCUSSION THREAD

**Discussion by rohith20-source** [BlockySite/community#13] [Q&A] [answered]
![Screenshot (166)](https://user-images.githubusercontent.com/91200067/188533622-bc2466cd-db6a-4fd9-9971-74f6a44b27e0.png)
[https://docs.blockysite.com/GithubApp/#permissions](url)

Well upon opening it's showing the page not found. And its says as [ Please contact the owner of the site that linked you to the original URL and let them know their link is broken.]

**Comment by shiva-sandupatla** [comment, marked as answer]
Hey, @rohith20-source.

Thanks for raising the issue. This issue has been resolved now. please have a look and let me know if you face any other issues.

**Reply by rohith20-source** [reply]
All ok !
